src/backend/TrafficCourts/Citizen.Service/Validators/ValidationMessages.cs

```diff
@@ -15,4 +15,5 @@ public static class ValidationMessages
     public static readonly string ViolationDateFutureInvalid = @"Violation Date must not be in the future. Read '{0}'.";
     public static readonly string ViolationDateGT30Days = @"Violation Date '{0}' is more than 30 days ago.";
     public static readonly string TimeInvalid = @"'{0}' is not a valid time.";
+    public static readonly string LowConfidenceError = @"Too many fields have a low confidence.";

```

Comment:
Just a note, you dont need to use "@" at the beginning of a string all the time. The "@" allows embedded newlines and changes how escape characters are handled... for example I might code:

```cs
string sql = @"
SELECT a, b
 FROM table
 WHERE id = @id";
```

Reply:
Thanks, was a little unclear on that.  I recall reading it was needed if there were any special characters in the string.  Not totally sure what is considered a special character ... these have '{', '}', and single quotes.